edk2toollib/uefi/edk2/path_utilities.py

```diff
@@ -120,60 +110,18 @@ def __init__(self, ws: str, package_path_list: Iterable[str],
             package_path_packages[package_path] = \
                 [p.parent for p in package_path.glob('**/*.dec')]
 
-        # Note: The ability to ignore this function raising an exception on
-        #       nested packages is temporary. Do not plan on this variable
-        #       being available long-term and try to resolve the nested
-        #       packages problem right away.
-        #
-        # Removal is tracked in the following GitHub issue:
-        # https://github.com/tianocore/edk2-pytool-library/issues/200
-        ignore_nested_packages = False
-        if "PYTOOL_TEMPORARILY_IGNORE_NESTED_EDK_PACKAGES" in os.environ and \
```

Comment:
If the variables are still being set after the change, it might be worth adding a warning message that the variables are now deprecated and ineffective. To help clean up consumer code and reduce confusion about what the variables might do if people see them still sitting around.

Reply:
I have added a warning that they are no longer used, and to remove them from your environment.

Thanks!